www/web-proxy-useracl/src/opnsense/mvc/app/controllers/OPNsense/ProxyUserACL/Api/SettingsController.php

```diff
@@ -74,7 +74,11 @@ public function addACLAction()
             $result = array("result" => "failed", "validations" => array());
             $mdlProxyUserACL = $this->getModel();
             $post = $this->request->getPost("ACL");
-            $post["Hex"] = $this->strToHex($post["Name"]);
+
+            if ($post["Name"])
```

Comment:
PSR1/2 style mandates curly braces everywhere

Reply:
PS: If source is the new field, why not test for this? makes the change easier to read